components/air_gapped_self_hosted_purchase_modal/index.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+import {useDispatch} from 'react-redux';
+import {FormattedMessage} from 'react-intl';
+
+import {CloudLinks, ModalIdentifiers} from 'utils/constants';
+import {closeModal} from 'actions/views/modals';
+import CreditCardSvg from 'components/common/svg_images_components/credit_card_svg';
+
+import './content.scss';
+
+export default function AirGappedSelfHostedPurhcaseModal() {
+    const dispatch = useDispatch();
+
+    const closeAirGappedModal = () => {
+        dispatch(closeModal(ModalIdentifiers.AIR_GAPPED_SELF_HOSTED_PURCHASE));
+    };
+
+    return (
+        <Modal
+            onHide={closeAirGappedModal}
+            show={true}
+            className='air-gapped-purhcase-modal'
+        >
+            <Modal.Body>
+                <div className='content'>
+                    <CreditCardSvg
+                        height={350}
+                        width={350}
+                    />
+                    <span id='air-gapped-modal-title'>
+                        <FormattedMessage
+                            id={'air_gapped_purchase_modal.title.purchase_through_portal'}
```

Comment:
I'm unsure if these ids are okay or not.

Reply:
Yeah, both ids look good to me. We're not super consistent in our naming and casing for i18n string ids, that form you used (underscores, and namespaced with `.`s) is the most common format. If you want, I've been namespacing things underneath `self_hosted_signup`. I think that would work here, too.

Reply:
Gotcha, I'll make this change before cherry-picking.